pkgs/applications/editors/vis/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, unzip, pkgconfig, makeWrapper, ncurses, libtermkey, lpeg, lua }:
+
+stdenv.mkDerivation rec {
+  name = "vis-${version}";
+  version = "git-2016-04-15";
+
+  src = fetchurl {
```

Comment:
Note that `fetchFromGitHub` is a convenient short-hand for this idiom, but it uses `fetchzip` under the hood which provides a stable hash (by computing it over the contents of the archive instead of the archive itself, which github likes to change every now and then, resulting in hash mismatches).